README.md

```diff
@@ -27,17 +26,19 @@ Despite example shows swap of ETH/ERC20 this approach will work also for ETH/ERC
 1. Run `docker-compose build`.
```

Comment:
Comment for the line above: it seems we don't need .env file anymore.

Reply:
> I got the error after following the environment setup guide from Readme:

> Comment for the line above: it seems we don't need .env file anymore.


So without `.env` you can execute hardhat?
I just dont have such error with and without `.env`. May be, because I use `docker compose` version.

Reply:
@artemii235 could you tell what docker version do you use? can you run newer docker compose commands please, Im just curious:
```
docker compose build
docker compose up -d
docker compose exec workspace yarn
docker compose exec workspace npx hardhat test
```

I would like to use Docker Compose V2 commands in readme.

Reply:
> I just dont have such error with and without .env. May be, because I use docker compose command version.

Comments about .env file and failure to run test commands are unrelated. .env contains variables that are no longer used, so the `.env.empty` file and `env_file` section in `docker-compose.yml` can be removed.

Reply:
`npx` was missed in run hardhat test command, fixed it. 
removed env file and env in commands (but left `.env` in `gitignore` just in case)